projects/1.16/assets/tinkers-construct/tconstruct/book/zh_cn/movingfluids/intro.json

```
@@ -0,0 +1,11 @@
+{
+  "title": "流体的转移",
+  "image": {
+    "file": "tconstruct:book/images/movingfluids.png"
+  },
+  "text": [
+    {
+      "text": "会用熔化炉后你可能会问：“但是Thruul，我怎么把那些材料弄出来呢？”你提出了个好问题，工匠！"
```

```suggestion
      "text": "会用熔化炉后你可能会问：“但是Thruul，我怎么把那些材料弄出来呢？”这是个好问题！"
```
这里句尾加称呼是否保留需要考虑一下。这个表达接近外国人的表达习惯，中文里一般不会这么说吧。

---

本来是打算用“你发现了盲点，华生”来凑合的
不过的确不符合中文习惯